include/pthread.h

```diff
@@ -333,25 +333,46 @@ typedef struct pthread_mutex_s pthread_mutex_t;
 #  endif
 #endif
 
+#ifndef CONFIG_PTHREAD_MUTEX_DEFAULT_PRIO_INHERIT
```

Comment:
let use postive logic
```suggestion
#ifdef CONFIG_PTHREAD_MUTEX_DEFAULT_PRIO_INHERIT
```